src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -881,5 +884,38 @@ object Helpers extends SparkSessionWrapper {
 
   }
 
+  def pipReport(etlDB: String = "overwatch_etl", detailed: Boolean = false):Dataset[Row] = {
+    try{
+      spark.catalog.getTable(s"${etlDB}.pipeline_report")
+      logger.log(Level.INFO, s"Overwatch has being deployed with  ${etlDB}")
+    }catch {
+      case e: Exception =>
+        val msg = s"Overwatch has not been deployed with  ${etlDB}"
+        logger.log(Level.ERROR, msg)
+        throw new BadConfigException(msg)
+    }
+
+    val pipReport = if (detailed) {
+      println("Report Extracted from pipeline_report")
+      table(s"${etlDB}.pipeline_report")
+    }else{
+      println("Report Extracted from pipReport")
+      table(s"${etlDB}.pipReport")
+    }
 
+    pipReport
+      .orderBy('Pipeline_SnapTS.desc,'moduleID)
+      .withColumn("fromTSt", from_unixtime('fromTS.cast("double") / lit(1000)).cast("timestamp"))
+      .withColumn("untilTSt", from_unixtime('untilTS.cast("double") / lit(1000)).cast("timestamp"))
```

Comment:
let's change the names to be a little more user friendly for the timestamps, such as "from_time", "until_time".

Let's move the columns in the simple view the front and move the rest of the columns to the end so that it's easier to read.

A select statement with appropriately ordered columns by importance will be very good here. You can either use select statement or df.moveColumnsToFront(...)